datayoga/blocks/filter/block.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from typing import Any
+
+from datayoga.block import Block as DyBlock
+from datayoga.blocks import expression
+from datayoga.context import Context
+
+logger = logging.getLogger(__name__)
+
+
+class Block(DyBlock):
+    def init(self):
+        logger.debug(f"Initializing {self.get_block_name()}")
+        self.language = self.properties["language"]
+        self.expression = expression.compile(self.language, self.properties["expression"])
+
+    def run(self, data: Any, context: Context = None) -> Any:
```

Comment:
data: Any should be replaced with data: List[Dict[str, Any]] and return type from Any to List[Dict[str, Any]].

Reply:
done